.pipelines/vhd-builder-windows.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
       -e BUILD_ID=$(Build.BuildId) \
       -e BUILD_NUMBER=$(Build.BuildNumber) \
       -e CONTAINER_RUNTIME=$(CONTAINER_RUNTIME) \
+      -e WINDOWS-SERVER-VERSION=2019 \

```

Comment:
should 2019 be an environment variable so it can be configured on the job?

Reply:
Possibly. We still need to have a discussion around what images aks-engine should actually be producing. If we want to produce more than just Windows Server 2019 + Docker I think I would rather just update the pipepline to build all the images as different stages and avoid the need to trigger multiple pipelines.
I'm hoping we can resolve that with a different PR tho.